runbotics/runbotics-scheduler/src/websocket/gateway/bot.gateway.ts

```diff
@@ -105,4 +105,11 @@ export class BotWebSocketGateway implements OnGatewayDisconnect, OnGatewayConnec
         this.botLogService.writeLogsToFile(client.bot.id, logs);
         this.logger.log(`<= Success: logs from bot ${installationId} saved`);
     }
+
+    private updateProcessInstanceEvent = async (bot: IBot, processInstanceEvent: IProcessInstanceEvent) => {
+        const installationId = bot.installationId;
+        this.logger.log(`=> Updating process-instance-event (${processInstanceEvent.executionId}) by bot (${installationId}) | step: ${processInstanceEvent.step}, status: ${processInstanceEvent.status}`);
+        await this.botProcessEventService.updateProcessInstanceEvent(processInstanceEvent, bot);
+        this.logger.log(`<= Success: process-instance-event (${processInstanceEvent.executionId}) updated by bot (${installationId}) | step: ${processInstanceEvent.step}, status: ${processInstanceEvent.status}`);
+    };
```

Comment:
hmm, I believe that `this` keyword does not work in arrow functions. i suggest to write it like a typical class method